_security/access-control/permissions.md

```diff
@@ -110,7 +110,7 @@ index_permissions:
 
 ### Verifying system index access
 
-You can use the [CAT indices]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-indices/) operation to see all indexes associated with any index pattern in your permissions configuration and verify that the permissions provide the access you intended. For example, if you want to verify a permission that includes system indexes beginning with the prefix `.kibana`, you can run the `GET /_cat/indices/.kibana*` call to return all indexes associated with that prefix.
+You can use the [CAT indexes]({{site.url}}{{site.baseurl}}/api-reference/cat/cat-indices/) operation to see all indexes associated with any index pattern in your permissions configuration and verify that the permissions provide the access you intended. For example, if you want to verify a permission that includes system indexes beginning with the prefix `.kibana`, you can run the `GET /_cat/indices/.kibana*` call to return all indexes associated with that prefix.
```

Comment:
Let's see how @kolchfa-aws responds about the API rename.

Reply:
Ok, I will wait for @kolchfa-aws input before updating the PR

Reply:
Let's change this one back to indices. @BigSamu

Reply:
Reversed the change!